database/postgresql/generateDishes.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const faker = require('faker');
+
+const randomCuisine = () => {
+  const cusines = ['italian', 'american', 'chineese', 'japanese', 'canadian'];
+  const randomIndex = Math.floor(Math.random() * (cusines.length - 1));
+  return cusines[randomIndex];
+};
+
+const numberOfRestaurants = 5000000;
+
+const createDishes = (numRestaurants) => {
+  const writer = csvWriter();
+  writer.pipe(fs.createWriteStream('./csv/dishes.csv'));
+  let dishCount = 0;
+  for (let i = 1; i < numRestaurants; i++) {
+    if (i % (Math.floor(numRestaurants / 10)) === 0) {
```

Comment:
Nice job implementing progress reports!

Reply:
Not sure, it updates about every 10% complete.